docs/UserGuide.adoc

```diff
@@ -46,132 +40,239 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `find b/BOOK_TITLE`, `BOOK_TITLE` is a parameter which can be used as `find b/Harry Potter`.
+* Items in square brackets are optional e.g `b/BOOK_TITLE [g/GENRE]` can be used as `b/Harry Potter g/fiction` or as `b/Harry Potter`.
+* Items with `...`​ after them can be used multiple times including zero times e.g. `[g/GENRE]...` can be used as `{nbsp}` (i.e. 0 times), `g/fiction`, `g/fiction g/adventure` etc.
+* Parameters can be in any order e.g. if the command specifies `b/BOOK_TITLE a/AUTHOR`, `a/AUTHOR b/BOOK_TITLE` is also acceptable.
 ====
 
-=== Viewing help : `help`
+=== Using Normal Mode
 
-Format: `help`
+The Normal Mode is for librarians to manage the books in the library. Commands in normal mode are not specific to any borrowers. The Normal Mode also provides some utility commands to help the librarian use Liberry. Commands that are specific to Serve Mode cannot be used while in Normal Mode.
 
-=== Adding a book: `add`
+==== Viewing help : `help`
 
-Adds a book to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Opens the help window, which includes a list of possible commands +
+Format: `help`
 
-[TIP]
-A book can have any number of genres (including 0)
+==== Locating books by title: `find`
 
-Examples:
+Shows all relevant books found. +
+Format: `find [b/BOOK_TITLE] [a/AUTHOR] [g/GENRE]... [sn/BOOK_SN] [-overdue] [-loaned] [-available]`
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+****
+* The search is case insensitive. e.g `harry` will match `Harry`
+* Only full words will be matched e.g. `Har` will not match `Harry`
+* Books matching at least one keyword will be returned (i.e. `OR` search). e.g. `harry` will return both `Harry Potter` and `Harry Botter`
+* `[b/BOOK_TITLE]` will be used to search through book titles
+* `[a/AUTHOR]` will be used to search through or filter by authors
+* `[g/GENRE]` will be used to search through or filter by genre
+* `[sn/BOOK_SN]` will be used to search through book serial numbers
+* Only 1 of the following 3 flags can be used
+* `[-overdue]` will only show overdue books
+* `[-loaned]` will only show loaned books
+* `[-available]` will only show available books
+//* Adding one of the following will sort the books such that:
+//* `[-popular]` will list the top 10 most borrowed books
+//* `[-new]` will list the 10 newest books
+//* `[-rated]` will list the top 10 highly rated books
+//* `[NUMBER]` will list this NUMBER of books instead
+****
 
-=== Listing all books : `list`
+Examples:
 
-Shows a list of all books in the address book. +
-Format: `list`
+* `find b/Animal Farm a/George Orwell` +
```

Comment:
I used prefix t for Title (t/Animal Farm) since it felt more specific, if confirm using b then I will change

Reply:
Thanks for the highlight. Will change to t/TITLE